app/containers/ChangeEmail/actions.js

```diff
@@ -107,3 +106,18 @@ export function changeEmailErr(changeEmailError) {
     changeEmailError,
   };
 }
+
+export function getEmailAddress(address) {
+  return {
+    type: GET_EMAIL_ADDRESS,
+    address,
+  };
+}
+
+export function getEmailAddressSuccess(email, isSubscribed) {
```

Comment:
why get? It's set

add action for failed

Reply:
done: https://github.com/peeranha/peeranha-web/pull/1278/commits/94afbcc0fd515b56fbd935c9a160f522e4fc9d68